src/Ray.php

```diff
@@ -541,6 +547,16 @@ public function sendRequest($payloads, array $meta = []): self
             // In WordPress this entire package will be rewritten
         }
 
+        if (self::rateLimit()->isMaxReached()) {
+            // @todo call ray once to notify the user has reached his limit
```

Comment:
Use `$this->sendCustom('Rate limit has been reached...', 'Rate limit')->red();` here

Reply:
This will end up in an infinite loop

Reply:
Why? Wouldn't there be a way to force this without ending in an infinite loop?